### PR TITLE
Openapi > ModelEndpoint > Normalize & Add responses

### PR DESCRIPTION
**Pull request by flyusfly**
- [x] - ModelEndpoint.get_collection() - Add a 401 response
- [x] - ModelEndPoint.get_resource() > Normalize & Add responses
- [x] - ModelEndPoint.post_resource() > Normalize & Add responses
- [x] - ModelEndPoint.post() > Normalize & Add responses
- [x] - ModelEndPoint.patch() > Normalize & Add responses
- [x] - ModelEndPoint.put() > Normalize & Add responses
- [x] - ModelEndPoint.delete() > Normalize & Add responses
- [x] - ModelEndpoint > Add a schema description for parameters identifiers
- [x] - ModelEndpoint > Simplify api identifier parameters

### CODE REVIEW

File: ban/http/api.py [hunk 1]
```diff
@@ -170,15 +172,22 @@ def get_resource(self, identifier):
 
         parameters:
             - $ref: '#/parameters/identifier'
+              name: identifier
```
Comment (flyusfly) [inline ban/http/api.py]: One solution is to put a static definition into schema.py :
```
    'parameters': {
        'identifier': {
            'name': 'identifier',
            'in': 'path',
            'type': 'string',
            'required': True,
            'description': 'resource identifier'
        }
```

for each ban/http/api.py properties, we just have to define :
```
        parameters:
            - $ref: '#/parameters/identifier'
```

What do you think about this?

Comment (yohanboniface) [inline ban/http/api.py]: isn't this already the case?

Comment (flyusfly) [inline ban/http/api.py]: Not really, we have no description in schema.py for parameters identifier.
This is more precise...

Comment (flyusfly) [inline ban/http/api.py]: U'll be :)

**Comment by flyusfly** [comment]
ModelEndpoint > Simplify api identifier parameters done.

**Pull request merged**